include/can/core/messages.hpp

```diff
@@ -355,10 +357,12 @@ struct MoveCompleted : BaseMessage<MessageId::move_completed> {
 struct EncoderPositionResponse
     : BaseMessage<MessageId::encoder_position_response> {
     int32_t encoder_position;
+    uint8_t position_flags;
```

Comment:
can we make this a `PositionResponse` and have both the stepper and the encoder position?

Reply:
Added this. The hardware interface now caches the steps and gets refreshed by the interrupt handler... I tried moving the position tracker completely, but turns out that ARM doesn't support 8-byte atomics 🙃